cloudsync/sync/state.py

```diff
@@ -916,7 +942,7 @@ def _mark_changed(self, side, ent):
             ent[side].changed = self._last_changed_time + 0.001
         self._last_changed_time = ent[side].changed
         log.debug("change time for side %s is %s", side, ent[side].changed)
-        assert ent in self._changeset
+        # assert ent in self._changeset
```

Comment:
can probably just take this out if it's not relevant anymore

Reply:
done